api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onEditorPreparing.md

```diff
@@ -66,6 +66,8 @@ The editor's value.
 The editor's width; equals **null** for all editors except for those whose **parentType** equals *"searchPanel"*.
 
 ---
+[note] In this function, you can change only **cancel**, **editorName**, **editorOptions**, and **updateValueTimeout**. All the rest parameters are read only.
```

Comment:
This is not how we should convey this information.

@viterobk Can we implement a `readOnly` attribute for function parameters?

Reply:
Need to research

Reply:
How should I say that everything but these 4 parameters are only for read-only access?

Reply:
You shouldn't. This should be dealt with by developers. My idea is to implement a `readOnly` attribute for function parameter fields. Developers would add this attribute to read-only fields, and DocGen would automatically update the .md-files. No input from technical writers required.

Reply:
Ok I am passing this info to dev team. If they approve your idea, I will close this PR.

Reply:
That's why I mentioned viterobk. He is the dev team.